### PR TITLE
Add benchmark script

### PR DESCRIPTION
**Pull request by liya2017**
add benchmark script

### CODE REVIEW

File: ckb-bench/script/benchmark.sh [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+            ;;
+        "build")
+            rust_build
+            ;;
+        "terraform")
+            terraform_apply
+            ;;
+        "ansible")
+            ansible_deploy_ckb
+            ansible_wait_ckb_synchronization
```
Comment (keroro520) [inline ckb-bench/script/benchmark.sh:188]: `s/synchronization/benchmark/g`

File: ckb-bench/script/benchmark.sh [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+function rust_build() {
+    git -C $JOB_DIRECTORY clone \
+        --branch develop \
+        --depth 1 \
+        https://github.com/nervosnetwork/ckb.git
+
+    cd $JOB_DIRECTORY/ckb
+    ls -al
+    pwd
```
Comment (keroro520) [inline ckb-bench/script/benchmark.sh:158]: remove debug code

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: ckb-bench/script/benchmark.sh (hunk 1 above)
Comment (liya2017) [reply ckb-bench/script/benchmark.sh]: done ,thanks

File: ckb-bench/script/benchmark.sh (hunk 2 above)
Comment (liya2017) [reply ckb-bench/script/benchmark.sh]: done. thanks.

**Review by keroro520 — APPROVED** [review]

**Pull request merged**